### PR TITLE
[VyOS: ]all modules updated with FQCN and new doc changes

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/vyos_firewall_global.py [hunk 1]
```diff
@@ -430,30 +428,22 @@
...
-  set firewall receive-redirects 'disable'
-  set firewall send-redirects 'enable'
-  set firewall source-validation 'strict'
-  set firewall state-policy established action 'accept'
-  set firewall state-policy established log 'enable'
-  set firewall state-policy invalid action 'reject'
-  set firewall syn-cookies 'enable'
-  set firewall twa-hazards-protection 'enable'"
+  vyos.vyos.vyos_firewall_global:
+    running_config: set firewall all-ping 'enable' set firewall broadcast-ping 'enable'
```
Comment (NilashishC) [inline plugins/modules/vyos_firewall_global.py:432]: This is not readable anymore. Can you see if running black (19.3b0) with line-length 79 fixes it?

File: plugins/modules/vyos_firewall_interfaces.py [hunk 2]
```diff
@@ -1013,16 +1012,13 @@
...
-      "set interfaces ethernet eth1 firewall in name 'INBOUND'
- set interfaces ethernet eth1 firewall out name 'OUTBOUND'
- set interfaces ethernet eth1 firewall local name 'LOCAL'
- set interfaces ethernet eth1 firewall local ipv6-name 'V6-LOCAL'
- set interfaces ethernet eth2 firewall in name 'INBOUND'
- set interfaces ethernet eth2 firewall out name 'OUTBOUND'
- set interfaces ethernet eth2 firewall local name 'LOCAL'
- set interfaces ethernet eth2 firewall local ipv6-name 'V6-LOCAL'"
+  vyos.vyos.vyos_firewall_interfaces:
+    running_config: set interfaces ethernet eth1 firewall in name 'INBOUND' set interfaces
```
Comment (NilashishC) [inline plugins/modules/vyos_firewall_interfaces.py:1016]: Same as above.

File: plugins/modules/vyos_firewall_rules.py [hunk 3]
```diff
@@ -1405,16 +1402,13 @@
...
-      "set firewall group address-group 'inbound'
- set firewall name Downlink default-action 'accept'
- set firewall name Downlink description 'IPv4 INBOUND rule set'
- set firewall name Downlink rule 501 action 'accept'
- set firewall name Downlink rule 501 description 'Rule 501 is configured by Ansible'
- set firewall name Downlink rule 501 ipsec 'match-ipsec'
- set firewall name Downlink rule 502 action 'reject'
- set firewall name Downlink rule 502 description 'Rule 502 is configured by Ansible'
- set firewall name Downlink rule 502 ipsec 'match-ipsec'"
+    running_config: set firewall group address-group 'inbound' set firewall name Downlink
```
Comment (NilashishC) [inline plugins/modules/vyos_firewall_rules.py:1405]: Same as above.

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**